wgsl/index.bs

```diff
@@ -916,7 +907,7 @@ TODO: Add links the eventual memory model descriptions.
     };
 
     [[group(0), binding(0)]]
-    var<storage> x: [[access(read_write)]] S;
+    var<storage,read_write> x: S;
```

Comment:
No matter how I look at it, the fact that we have the access qualifier here, and it only makes sense for `storage` class, seems awkward :/

Reply:
?

It's the only one with two choices, which is what makes it special. 
And since there is a choice, the programmer has to say something to distinguish the cases.